### ISSUE TITLE
Class hall druid 

### ISSUE THREAD

**Issue by Derifench** [LegionIsrael/LegionIsraelBugTracker#44]
I stood in dalaran for about 2 hours, then I ran along the dalaran near the building of the alliance, applied different forms, and a tauren appeared next to me (He ran somewhere from the textures), and the ego who beat (I didn’t see because the whole client), he had a task that I passed in the moon glade, and I was given a spell portal, with which I got into the grove of dreams (translator, sorry)

**Comment by Derifench** [comment]
It seems to me that this npc ran to me all the time, since it was based on the plot, all the locations are in Azeroth through which I traveled for a long time, and since I stood in one place for a long time he got to me

**Comment by Derifench** [comment]
https://ibb.co/d7wFoe

**Comment by LegionIsrael** [comment]
yeah, it should be instant on every druid, we'll hotfix it to add the spell already on every newly created druid toon.

Thanks
